mlrun/runtimes/package/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
why directory name `package` sounds quite too generic

Reply:
I'll write packagers classes (hence the name package) to `unpack` (parse `DataItem` to type hint) and `pack` (log an object to MLRun using log hint) so users will be able to write there own parsing and logging logic and add it to MLRun to have fully dynamic objects parsing and logging between functions.